scripts/DEMO-2-BPs.md

```diff
@@ -0,0 +1,85 @@
+# Running a local example with Two Business Partners
+
+## Build and run the universal DID resolver
+
+First clone and build the sovrin resolver (for bcovrin test):
+
+```bash
+git clone https://github.com/ianco/uni-resolver-driver-did-sov.git
+cd uni-resolver-driver-did-sov
+docker build -f ./docker/Dockerfile . -t universalresolver/driver-did-sov
+```
+
+Now run the universal resolver (the only profile is for our bcovrin resolver)
+
+```bash
+git clone https://github.com/ianco/universal-resolver.git
+cd universal-resolver
+docker pull universalresolver/uni-resolver-web:latest
+docker-compose -f docker-compose.yml up
+```
+
+Once the uni resolver is running you should be able to curl a DID on the ledger, for example:
+
+```bash
+curl -X GET http://localhost:8080/1.0/identifiers/did:sov:MTMagrM95WXayAHfwTNY17
+```
+
+## Run the first BP Agent
+
+```bash
+git clone https://github.com/ianco/business-partner-agent.git
+cd business-partner-agent
+docker build -t ghcr.io/hyperledger-labs/business-partner-agent:local .
```

Comment:
The build step shouldn't be necessary as the image is being pulled via BPA_DOCKER_IMAGE which is set to the one from GitHub